src/main/java/com/minecolonies/entity/pathfinding/PathNavigate.java

```diff
@@ -263,6 +279,35 @@ else if (entity.isInWater())
         }
     }
 
+    @Override
+    protected void pathFollow()
+    {
+        int curNode = currentPath.getCurrentPathIndex();
+        int curNodeNext = curNode + 1;
+        if (curNodeNext < currentPath.getCurrentPathLength())
+        {
+            PathPointExtended pEx = (PathPointExtended) currentPath.getPathPointFromIndex(curNode);
+            PathPointExtended pExNext = (PathPointExtended) currentPath.getPathPointFromIndex(curNodeNext);
+
+            //  If current node is bottom of a ladder, then stay on this node until
+            //  the entity reaches the bottom, otherwise they will try to head out early
+            if (pEx.isOnLadder && pEx.ladderFacing == EnumFacing.DOWN &&
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '&&' should be on a new line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AOperator_Wrap)